python/operation_types.py

```diff
@@ -20,6 +26,7 @@
     "DIVIDE": [("int", "int", "int"), ("float", "float", "float")],
     "SHIFTL": [("int", "int", "int")],
     "SHIFTR": [("int", "int", "int")],
+    "IN": [("char", "str", "bool"), ("str", "str", "bool"), (in_generic, n_list_type.with_typevars([in_generic]), "bool"), (in_map_generic, n_map_type.with_typevars([in_map_generic, in_value_generic]), "bool")],
```

Comment:
Epic! For maps, this'll check if the given value is a *key* in the map; just making sure.

Reply:
Yes